WcaOnRails/app/models/competition.rb

```diff
@@ -15,18 +15,48 @@ class Competition < ActiveRecord::Base
   has_many :organizers, through: :competition_organizers
   has_many :media, class_name: "CompetitionMedium", foreign_key: "competitionId", dependent: :delete_all
 
+  CLONEABLE_ATTRIBUTES = %w(cityName
+                            countryId
+                            information
+                            eventSpecs
+                            venue
+                            venueAddress
+                            venueDetails
+                            website
+                            latitude
+                            longitude
+                            contact
+                            remarks
+                            use_wca_registration
+                            guests_enabled).freeze
```

Comment:
What about grouping related fields? This would save some space, but I'm not sure what's better =)
```ruby
CLONEABLE_ATTRIBUTES = %w(
  countryId cityName
  venue venueAddress venueDetails
  contact website information remarks
  eventSpecs
  latitude longitude
  use_wca_registration
  guests_enabled
).freeze
UNCLONEABLE_ATTRIBUTES = %w(
  id name cellName
  year month day
  endMonth endDay
  showAtAll isConfirmed
  registration_open registration_close
  results_posted_at
).freeze

```